test_unit.py

```diff
@@ -4060,6 +4060,13 @@ def test_array_static_encoding(self):
                     "00 00 00 00 00 00 00 01 00 00 00 00 00 00 00 02 00 00 00 00 00 00 00 03"
                 ),
             ),
+            (
```

Comment:
Added a unit test that was helpful during development. I don't see the harm of keeping.